### ISSUE TITLE
can CAT (client address translation) be combined with on-demand

### ISSUE THREAD

**Issue by cagney** [libreswan/libreswan#942]
assign_holdpass(), triggered by acquire, has a call to install a CAT kernel policy (along side the hold).
Tests show it never happening.

**Comment by letoams** [comment]
Yes it should have a hold for the remote ip, as CAT is part of OE that all happens on demand

Sent using a virtual keyboard on a phone
